packages/fuel-indexer/src/executor.rs

```diff
@@ -90,23 +90,18 @@ pub fn run_executor<T: 'static + Executor + Send + Sync>(
         .unwrap_or(config.fuel_node.to_string());
 
     // Where should we initially start when fetching blocks from the client?
-    let mut cursor = executor.manifest().start_block().map(|x| {
```

Comment:
This is incorrect. We shouldn't be mutating any inputs. If the user want's to provide a certain `start_block` they should do that. If the user provides an invalid `start_block` just let it error out.